catalogue/utils.py

```diff
@@ -43,3 +47,15 @@ def prune_files(files, dir):
     list of str
     """
     return [f for f in files if dir != os.path.basename(os.path.dirname(f))]
+
+
+
+def read_config_file(config_file):
+    with open(config_file) as f:
+        config_data = yaml.load(f, Loader = yaml.FullLoader)
+    return config_data
+
+
+def dictionary_printer(dict):
+    for key, value in dict.items():
+        print('{}:{}'.format(key, value))
```

Comment:
I can be kind of a stickler for typesetting detail, so I would suggest adding a space after the colon for clarity.